src/main/java/seedu/guest/logic/commands/AddCommand.java

```diff
@@ -29,14 +30,16 @@ public class AddCommand extends Command {
             + PREFIX_DATE_RANGE + "DATE_RANGE "
             + PREFIX_NUMBER_OF_GUESTS + "NUMBER_OF_GUESTS "
             + PREFIX_IS_ROOM_CLEAN + "IS_ROOM_CLEAN "
+            + PREFIX_REQUEST + "REQUEST(Optional) "
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ROOM + "05-73 "
             + PREFIX_DATE_RANGE + "13/09/22 - 15/09/22 "
             + PREFIX_NUMBER_OF_GUESTS + "1 "
-            + PREFIX_IS_ROOM_CLEAN + "yes";
+            + PREFIX_IS_ROOM_CLEAN + "yes"
+            + "(Optional: " + PREFIX_REQUEST + "Apply for room service" + ")";
```

Comment:
Should we remove the Optional: and brackets here to just put the request? Also "Apply for room service" is a weird request should we change it to something more normal?